Src/Witsml/Data/WitsmlClientCapabilities.cs

```diff
@@ -44,7 +44,7 @@ public string ToXml()
         }
     }
 
-    public class ClientContactInformation
+    public class ContactInformation
```

Comment:
I think that renaming classes in the Witsml project counts as a "breaking change", due to other projects that use the Witsml nuget package having to rename the class on their end.

Reply:
PR marked as "breaking change".